about.html

```diff
@@ -0,0 +1,136 @@
+          <img id="logo-btn" src="/bg-img/logo.png" alt="logo-icon">
+          </a>
+        <div id="nav-pages">
+          <a class="pop-window" href="/about.html">About</a>
+          <a class="pop-window" href="#about">Program</a>
+          <a class="pop-window" href="#footer"> Join</a>   
+          <a class="pop-window" href="#footer"> Sponsors</a>  
+          <a class="pop-window" href="#footer"> News</a>  
+        </div>
+        <div id="contact"> 
+          <button id="contact-btn" > Contact </button>
+        </div>
+      </div>
+    </nav>
+   <div id="about-bg"> 
+    <div class="title-about">
+     <h1> GET READY FOR THE ULTIMATE FESTIVAL WINTER 2023 </h1>
+    </div>
+    <div class="paragraph" id="about-p"> Lolla takes over historic Grant Park in Downtown Chicago, alongside the beautiful Chicago skyline and the shores of Lake Michigan. Take advantage of interactive areas 
+     with giveaways, merch and more!</div>
+    <div class="date" >SAVE THE DATE! MARCH 2-5-2023
+     <p>@ Chicago, Illinois, USA.</p>
+    </div>
+   </div>
+  </div>
+ </section>
+
+ <!-- SECTION 2 -->
+
+ <section id="section-2">
+  <div id="about-bg-2"> 
+   <div class="title-about-2">
+    <h1> ULTRA-PALLOZA LOGO <div id="program-separator-line" class="guide-bar"></div></h1>
+   </div>
+   <div class="paragraph" id="about-p">
+    <p>
+     The ULTRA-PALLOZA logo was made by  street artist BAKSY in 2022
+    </p> 
+   </div>
+   <div id="about-img-2">
+    <img src="/bg-img/Captura de pantalla 2022-10-18 125031.png" alt="festival-img">
+   </div>
+  </div>
+ </section>
+
+ <!-- SECTION-3 -->
+
+ <section id="section-3">
+  <div id="about-bg-3"> 
+   <div class="title-about-2">
+    <h1> SEE THE PREVIOUS EDITON OF THE FESTIVAL <div id="program-separator-line" class="guide-bar"></div></h1>
+   </div>
+   <div class="paragraph" id="about-p"> ULTRA PALLOZA is a place to relax and meet new friends, listen to some cool indie bands and chill</div>
+   <div id="about-img-3">
+    <div id="red-img"> 
+     <h1> 2021</h1>
+     <h3>ULTRA-PALLOZA Buenos aires</h3>
+    </div>
+    <div id="red-img-2"> 
+     <h1> 2020</h1>
+     <h3> ULTRA-PALOZZA Quito</h3>
+    </div>
+   </div>
+  </div>
+ </section>
+
+ <!-- SECTION 4 -->
+
+ <section id="section-4">
+  <div id="about-bg-4"> 
+   <div id="title-footer">
+    <h3> Partner <div id="program-separator-line" class="guide-bar"></div></h3>
+   </div>
+   <div class="about-img-4">
+    <img src="/bg-img/partner1-removebg-preview.png" alt="">
+    <img src="/bg-img/partner2-removebg-preview.png" alt="festival-img">
+    <img src="/bg-img/partner3-removebg-preview.png" alt="festival-img">
+    <img src="/bg-img/partner44-removebg-preview.png" alt="">
+    <img src="/bg-img/partner5-removebg-preview.png" alt="">
+   </div>
+  </div>
+ </section>
+ <footer id="footer-sec">
+  <div id="footer-part">
+<img src="/bg-img/logo.png" alt="">
+<h3>2022 ULTRA-PALOZA. All rights reserved</h3>
+</div>
+</footer>
```

Comment:
> According to the original design ⬇
- [ ] Partners section should appear on `Mobile Version` only, so kindly remove it from the `Desktop version` 👍
- [ ] The `background-color` for the Footer section should be `dark` on `Desktop version` and `white` on `Mobile version`, kindly fix that 💯